diagrams/Systems-Theoretic-Safety-Process-v2.svg

```diff
@@ -24,8 +26,32 @@
         <dc:format>image/svg+xml</dc:format>
         <dc:type
            rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
-        <dc:title />
+        <dc:title>Systems Theoretic Safety Process</dc:title>
+        <cc:license
+           rdf:resource="http://creativecommons.org/licenses/by-nc-sa/4.0/" />
```

Comment:
Here the reference to by-nc-sa is not intended, you intended to refer to by-sa.

Reply:
Thanks Lukas - I've fixed this